SPEC.md

```diff
@@ -106,6 +127,12 @@ Success is indicated by a return code of zero and the following JSON printed to
 The result is returned in the same format as specified in the [configuration](#network-configuration).
 The specification does not declare how this information must be processed by CNI consumers.
 Examples include generating an `/etc/resolv.conf` file to be injected into the container filesystem or running a DNS forwarder on the host.
+The `interfaces` describes specific network interfaces the plugin created and their IP configuration.
+If the plugin does not use L2 addressing and thus MAC addresses are not meaningful, then an interface element's "mac" key may be omitted.
+The `sandbox` key must be provided for interfaces created or moved into a sandbox like a network namespace or a hypervisor/VM.  For container/namespace-based environments the `sandbox` key should be the full filesystem path to the network namespace of that sandbox. Hypervisor/VM-based plugins should return an ID unique to the virtualized sandbox the interface was created in.
+If the `CNI_IFNAME` variable is passed to the plugin, it must honor that name or return an error if it cannot.
+Each interface in the `interfaces` list should also return IP addressing information if present.
+Plugins must also return duplicate IP address information for the sandboxed interface in the top-level `ip4` and `ip6` keys.
```

Comment:
It needs to be specified which interfaces `ip4` and `ip6` belong to in regard to the ips lists embedded in the interfaces list.